internal/mods.json

```diff
@@ -102,12 +105,118 @@
             "serverlist": "",
             "contenttext": "Community made Visual Fixes for various issues with cosmetics, weapons, particles, visual effects,\n plus fixing Dragon’s Fury lag, opaque and glitchy Flamethrower flames, worse looking stock weapons, and more!",
             "install": {
+                "modname": "Ultimate TF2 Visual Fix Pack",
                 "type": "github",
                 "owner": "agrastiOs",
                 "name": "Ultimate-TF2-Visual-Fix-Pack",
                 "targetdirectory": "{tf2_dir}/tf/custom/",
                 "asset_index": 0
             }
+        },
+        {
+            "name": "mastercomfig",
+            "blurb": "mastercomfig",
+            "icon": "images/logos/modcirclelogos/mastercomfig_logo_transparent.svg",
+            "titleimage": "images/logos/modcirclelogos/mastercomfig_logo_transparent.svg",
+            "backgroundimage": "images/backgrounds/cp_badlands01.jpg",
+            "backgroundBlendMode": "multiply",
+            "bordercolor": "#038a7d",
+            "backgroundposX": "left",
+            "backgroundposY": "top",
+            "website": "https://mastercomfig.com/",
+            "github": "https://github.com/mastercomfig/mastercomfig",
```

Comment:
I would rather if only the website was linked, as the website appropriately links to everything else, including the GitHub. I don't really want normal users going on the GitHub as it can be confusing.